### ISSUE TITLE
Nested Backups Become Unnested

### ISSUE THREAD

**Issue by zx2c4** [restic/restic#910]
If I run: `restic backup / /mnt/storage/Archives`, when I then run `restic mount`, I expect to see:

```
/
  bin
  etc
  ...
  mnt
    storage
      Archives
```

Instead, I see:

```
/
  Archives
  bin
  etc
  ...
  mnt
```

**Comment by fd0** [comment]
I can see where you're coming from. This unfortunately is the result of a (bad) design decision early on, and I'm planning to rewrite the archiving code to correct this. It is all wrong, there are many special cases that aren't covered by the current approach, so this part of restic will be rewritten. I'm going to close this as a duplicate of #549.

**Comment by zx2c4** [comment]
@fd0 you mentioned that this issue was "fixed" in the release notes of 0.6.0, but reading this issue, it looks like you more simply just closed it as a duplicate of another one. Should that other one be closed too because this issue is actually fixed? Or is this issue not yet fixed? Wondering so I know whether or not I can make my scripts a bit more sane by removing to workaround...

**Comment by fd0** [comment]
Ah, I'm sorry for the confusion: It is not fixed yet.

I closed this issue because the underlying issue is tracked in #549, so this is a duplicate. For the extended changelog I'm using [github_changelog_generator](https://github.com/skywinder/github-changelog-generator), which (among others) lists all issues that were closed since the last release.

Now that you bring it up I can understand your confusion and I'll see what I can do about it. Thanks!
